pkg/tdex-market-loader/client.go

```diff
@@ -92,6 +101,51 @@ func (t *tdexMarketLoaderService) FetchBalance(
 		},
 	})
 	if err != nil {
+		if s, ok := status.FromError(err); ok {
+			if s.Code() == codes.Unavailable {
+				requestData, err := json.Marshal(MarketReq{
```

Comment:
It's totally fine to use the types from the proto stubs `tdexv1`, we don't need these ad hoc files.

Reply:
Done